### PR TITLE
feat(decline-registration): decline registration confirmation logic

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Decline Registration confirmation logic

## Why

Decline Registration confirmation functionality when user click on decline link from email

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by jjeroch** [comment]
recheck the branch please - as per the ticket this should be release branch related

File: src/components/pages/DeleteCompany/DeleteCompany.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```
Comment (oyo) [inline src/components/pages/DeleteCompany/DeleteCompany.scss:2]: 2024

File: src/components/pages/DeleteCompany/index.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```
Comment (oyo) [inline src/components/pages/DeleteCompany/index.tsx:2]: 2024

File: src/features/deleteCompany/deleteCompanyApiSlice.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```
Comment (oyo) [inline src/features/deleteCompany/deleteCompanyApiSlice.ts:3]: 2024 and remove BMW line

**Review by oyo — CHANGES_REQUESTED** [review]
update header dates

File: src/features/deleteCompany/deleteCompanyApiSlice.ts (hunk 3 above)
Comment (nidhigarg-bmw) [reply src/features/deleteCompany/deleteCompanyApiSlice.ts]: @oyo Updated all the headers

**Pull request merged**